### PR TITLE
fix(seed): updated basic javascript URLs for hints

### LINKED ISSUE #17416: Basic JavaScript - Record Collection: Broken Link
There's a broken link in this lesson: https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/basic-javascript/record-collection
The second line in the "Hints" section says
>Use bracket notation when [accessing object properties with variables](https://learn.freecodecamp.org/accessing-objects-properties-with-variables).
> You may refer back to [Manipulating Complex Objects](https://learn.freecodecamp.org/manipulating-complex-objects) Introducing JavaScript Object Notation (JSON) for a refresher. 

These links are broken, pointing to an error page (shown in screenshot).

* Browser Name: Mozilla Firefox 
* Browser Version: v60
* Operating System: Ubuntu
screenshot from 2018-06-03 23-58-12

### PR DESCRIPTION
**Pull request by bogadodiegoh**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [ ] Tested changes locally.
- [x] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #17416

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
@bogadodiegoh, thank you very much for submitting a PR for this. However, it seems like there's a slight issue with the URL you updated. Please take a look at my review.

File: seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json [hunk 1]
```diff
@@ -6807,7 +6807,7 @@
             "Si la propiedad <code>prop</code> es <code>\"tracks\"</code> y <code>value</code> no está en blanco, empuja (<em>push</em>) el valor <code>value</code> al final del vector <code>tracks</code>.",
             "Si el valor <code>value</code> está en blanco, elimina esa <code>prop</code>.",
             "Siempre retorna el objeto <code>collection</code> entero.",
-            "<strong>Nota</strong><br>No olvides usar <code>notación corchete</code> cuando <a href=\"accessing-objects-properties-with-variables\" target=\"_blank\">accedes a propiedades de objetos con variables</a>."
+            "<strong>Nota</strong><br>No olvides usar <code>notación corchete</code> cuando <a href=\"javascript-algorithms-and-data-structures/basic-javascript/accessing-objects-properties-with-variables\" target=\"_blank\">accedes a propiedades de objetos con variables</a>."
```
Comment (scissorsneedfoodtoo) [inline seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json]: This line, too:
`javascript-algorithms-and-data-structures/basic-javascript/accessing-object-properties-with-variables`

File: seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json [hunk 2]
```diff
@@ -6757,7 +6757,7 @@
         "If <code>prop</code> is <code>\"tracks\"</code> but the album doesn't have a <code>\"tracks\"</code> property, create an empty array before adding the new value to the album's corresponding property.",
         "If <code>prop</code> is <code>\"tracks\"</code> and <code>value</code> isn't empty (<code>\"\"</code>), push the <code>value</code> onto the end of the album's existing <code>tracks</code> array.",
         "If <code>value</code> is empty (<code>\"\"</code>), delete the given <code>prop</code> property from the album.",
-        "<strong>Hints</strong><br>Use <code>bracket notation</code> when <a href=\"accessing-objects-properties-with-variables\" target=\"_blank\">accessing object properties with variables</a>.",
+        "<strong>Hints</strong><br>Use <code>bracket notation</code> when <a href=\"javascript-algorithms-and-data-structures/basic-javascript/accessing-objects-properties-with-variables\" target=\"_blank\">accessing object properties with variables</a>.",
```
Comment (scissorsneedfoodtoo) [inline seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json]: Could you change this to the following:

`javascript-algorithms-and-data-structures/basic-javascript/accessing-object-properties-with-variables`

**Review by scissorsneedfoodtoo — CHANGES_REQUESTED** [review]

File: seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json [hunk 3]
```diff
@@ -6757,7 +6757,7 @@
...
         "Push is an array method you can read about on <a href=\"https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push\" target=\"_blank\">Mozilla Developer Network</a>.",
         "You may refer back to <a href=\"manipulating-complex-objects\" target=\"_blank\">Manipulating Complex Objects</a> Introducing JavaScript Object Notation (JSON) for a refresher."
```
Comment (scissorsneedfoodtoo) [inline seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json]: While you're at it, could you also change this link?:
`javascript-algorithms-and-data-structures/basic-javascript/manipulating-complex-objects`

Comment (scissorsneedfoodtoo) [inline seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json]: While you're at it, could you also update this link?:
`javascript-algorithms-and-data-structures/basic-javascript/manipulating-complex-objects`

**Comment by raisedadead** [comment]
@scissorsneedfoodtoo see https://github.com/freeCodeCamp/freeCodeCamp/pull/17471#issuecomment-395914805

**Comment by bogadodiegoh** [comment]
Updated broken links with review suggestions.

**Comment by scissorsneedfoodtoo** [comment]
@bogadodiegoh, thank you for making those changes! Everything looks great.

@raisedadead, this PR is ready to merge.

**Comment by scissorsneedfoodtoo** [comment]
@bogadodiegoh, thank you for your contribution! :+1: :+1: :+1: We all look forward to your next PR!

@raisedadead, this one has been merged.

**Pull request merged**